test/test_04_02.py

```diff
@@ -0,0 +1,17 @@
+from unittest.mock import patch
+
+import pytest
+
+from hm_1.tasks.task_04_02 import count_dots_on_i
+
+
+def test_count_dots_on_i():
+    with patch("requests.get") as fake:
+        fake.return_value = 59
+        assert count_dots_on_i("https://example.com/") == fake.return_value
```

Comment:
If you don't visit the internet you can use any URL: let's change this to `zzz` to be sure.